packages/desktop-client/src/components/manager/BudgetList.js

```diff
@@ -195,9 +195,10 @@ function File({ file, onSelect, onDelete }) {
 function BudgetTable({ files, onSelect, onDelete }) {
   return (
     <View
+      className="rabbit"
```

Comment:
?

Reply:
Whoops. My foolproof method for finding a div in devtools when the cryptic class names aren't helping...